doc/source/examples/dem/3d-small-scale-rotating-drum/3d-small-scale-rotating-drum.rst

```diff
@@ -0,0 +1,201 @@
+
+==================================
+Small Scale Rotating Drum (3D)
+==================================
+
+This example of Lethe-DEM simulates dry granular flow behaviour in a small scale rotating drum. The discrete element method (DEM) is responsible for describing the behaviour of particles.  More information regarding the DEM parameters are given in the Lethe-DEM documentation, i.e. `DEM parameters <../../../parameters/dem/dem.html>`_.
+
+
+Features
+----------------------------------
+- Solvers: ``dem_3d``
+- Three-dimensional problem
+- Rotational boundary
+- Load-balancing
+
+
+
+Locations of Files Used in the Example
+--------------------------------------------------
+
+ - Parameters file for particle insertion: ``/examples/dem/3d_small_scale_rotating_drum/packing_rotating_drum.prm``
+ - Parameters file for drum rotation: ``/examples/dem/3d_small_scale_rotating_drum/small_rotating_drum_dem.prm``
+
+
+
+Description of the Case
+-----------------------
+
+This example simulates a rolling regime in a small scale rotating drum. First, we use Lethe-DEM to fill the bed with 20000 particles. We enable check-pointing in order to write the DEM checkpoint files for the packing which will be used as the starting point of the DEM simulation of the rotating drum. The solver ``dem_3d`` is used to simulate the behaviour of dry granular flow within the rotating drum.
+
+
+
+Parameter File
+--------------
+
+Mesh
+~~~~~
+
+In this example, we choose a ``cylinder`` grid type to create a cylinder. Grid arguments are the radius of the cylinder (0.056 m) and half-length (0.051 m), respectively.  The grid is refined 3 times using the ``set initial refinement`` parameters. The ``expand particle-wall contact search`` is used in concave geometries to enable extended particle-wall contact search with boundary faces of neighbor cells for particles located in each boundary cell (for more details see `Rotating Drum example <../rotating-drum/rotating-drum.html>`_).
+
+.. code-block:: text
+
+    subsection mesh
+        set type                 				= dealii
+        set grid type      	     				= cylinder
+        set grid arguments       				= 0.056:0.051
+        set initial refinement   				= 3
+    end
+
+
+Packing Information
+~~~~~~~~~~~~~~~~~~~~
+
+An insertion box is defined inside the cylindrical domain, inserting 8000 particles every 0.5 seconds while the cylinder is at rest. It is important to note the size of the insertion box to make sure it is completely inside our geometry. Otherwise, particles will be lost during the insertion stage.
```

Comment:
Can you refer to another example that explains the insertion box setup? Maybe packing in a cylinder would be a good one.